bundles/core/org.eclipse.smarthome.core.voice/src/main/java/org/eclipse/smarthome/core/voice/VoiceManager.java

```diff
@@ -469,6 +470,26 @@ public TTSService getTTS() {
     }
 
     /**
+     * Retrieves a TTS service with the given id.
+     *
+     * @param id the id of the TTS service
+     *
```

Comment:
I don't know about a strong rule here, but I don't think we need to add empty lines between @param and @return here.
IIRC it is normally not used (e.g. http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#format) and the documentation is something that is consumed by a doc generator, so not need to add empty lines between tags at all.